### ISSUE TITLE
Add support for MySensors own Sensebender Gateway and Sensebender Micro boards

### ISSUE THREAD

**Issue by nutkracker** [mysensors/NodeManager#221]
Great project.  As topic.

**Comment by user2684** [comment]
Good idea. I guess this will not be super simple to implement but for sure is a great idea. Thanks!

**Comment by ghvader** [comment]
I second this request :)

**Comment by user2684** [comment]
> can you add support for the Si7021 sensor as found in sensebender micro?
> https://www.mysensors.org/hardware/micro

https://forum.mysensors.org/topic/6534/nodemanager/130

**Comment by nutkracker** [comment]
Thank you!
